pages/notes/easter-eggs-by-nicolas-keppens.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: "Easter Eggs by Nicolas Keppens"
+categories:
+  - name: films
```

Comment:
This is how categories are defined in frontmatter (a list of objects with the `name` key is required because of mustache templating limitations)